RoboFile.php

```diff
@@ -10,10 +10,16 @@ class RoboFile extends \Robo\Tasks
     /**
      * Run the Robo unit tests.
      */
-    public function test($args = "")
+    public function test($args = "", $options = ['coverage' => false])
     {
-        return $this->taskCodecept()
+        $taskCodecept = $this->taskCodecept()
             ->args($args);
+
+        if ($options['coverage']) {
+            $taskCodecept->coverageXml('build/logs/clover.xml');
```

Comment:
phpunit created it for me, but I could try adding build/logs/.gitkeep if codeception would be happier with it there. If codeception just calls through to phpunit, this probably isn't necessary.